ember-app/package.json

```diff
@@ -21,6 +21,7 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.0.0",
+    "ember-browserify": "1.1.8",
```

Comment:
so, I'd like to keep this in there because it's handy and awesome for bringing in npm installed libraries. 

Tho I didn't get a chance to use it, it's very helpful